src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,28 @@
+.stars {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(../images/star.svg);
```

Comment:
u can use just `background` for it
https://developer.mozilla.org/en-US/docs/Web/CSS/background